frontend/src/hooks/users.ts

```
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import type { User, LoginRequest } from '../types/users.ts';
+
+export const fetchUserSignUp = async (user: User) => {
+    try {
+        await axios.post('http://localhost/api/signup', user);
+    } catch (error: any) {
+        throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)